lib/services/ios-debug-service.ts

```diff
@@ -72,14 +74,15 @@ class IOSDebugService implements IDebugService {
         }).future<void>()();
     }
 
-    private emulatorDebugBrk(): IFuture<void> {
+    private emulatorDebugBrk(shouldBreak? : boolean): IFuture<void> {
```

Comment:
Forgot to mention that this is not a merge stopper, it's up to you to decide which is the approach you prefer :) :)